src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -9,35 +9,53 @@
 using Microsoft.Azure.WebJobs.Host;
 using Microsoft.Azure.WebJobs.Host.Executors;
 using Microsoft.Azure.WebJobs.Host.Listeners;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.WindowsAzure.Storage.Queue;
 
 namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
 {
     // TODO: leveraging singleton listener for now until the framework supports a generalized listener/queue mechanism.
     [Singleton(Mode = SingletonMode.Listener)]
     internal class ApiHubListener : IListener
     {
-        private IFolderItem _folderSource;
-        private ITriggeredFunctionExecutor _executor;
+        // Need to be kept in sync with the Antares Scale Controller
```

Comment:
Recommend removing this comment. Let's keep the extensions code independent of outside particulars.